tests/casp/filesystem_ro.pm

```diff
@@ -16,12 +16,8 @@ use testapi;
 
 sub run() {
     assert_script_run "! touch /should_fail";
-
-    # /etc is ro now but may be changed later
-    assert_script_run "! touch /etc/should_fail";
-
+    assert_script_run "touch /etc/should_succeed";
```

Comment:
There is only root in CaaSP, so I run select_console only once (to login). I can check that I am root, but I think it's not really needed.